ParserXML/Parser.cs

```diff
@@ -1,46 +1,74 @@
 ﻿using System;
```

Comment:
Structure of your class. I ask to put each class in separete cs with name of class.